src/ProgrammeScriptContainer/index.js

```diff
@@ -1,18 +1,36 @@
-import React, { useState } from 'react';
+import React, { useState, useRef } from 'react';
 import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
 import arrayMove from 'array-move';
 import { SortableContainer, } from 'react-sortable-hoc';
 import ProgrammeElements from './ProgrammeElements';
+import PreviewCanvas from '../PreviewCanvas';
 
 const ProgrammeScriptContainer = (props) => {
   const [ items, setItems ] = useState(props.items);
+  const [ timeOnClick, setTimeOnClick ] = useState();
+  const width = useState(150);
```

Comment:
does this make the video go narrow?
```suggestion
  const width = useState(300);
```